.gitignore

```diff
@@ -104,5 +104,8 @@ safepaths.realm
 # swift packages
 ios/COVIDSafePaths.xcworkspace/xcshareddata/swiftpm/Package.resolved
 
+# Firebase
+android/app/google-services.json.real
+android/app/google-services.json.mock
```

Comment:
I think we don't need these lines
We will replace the `google-services.json` file to distribute builds

Reply:
But I need to keep those files in my local repo because I can't use the HA script to get the real file (the HA script fails for me because I don't have credentials for it)